docs/dev-guide.md

```diff
@@ -589,58 +436,62 @@ WHERE
 - To write a callback-style function that internally uses async/await code, use this pattern:
 
   ```javascript
-  const util = require('util');
+  import util from 'node:util';
+
   function oldFunction(x1, x2, callback) {
-      util.callbackify(async () => {
-          # here we can use async/await code
-          y1 = await f(x1);
-          y2 = await f(x2);
-          return y1 + y2;
-      })(callback);
+    util.callbackify(async () => {
+      // here we can use async/await code
+      y1 = await f(x1);
+      y2 = await f(x2);
+      return y1 + y2;
+    })(callback);
   }
   ```
 
-- To write a multi-return-value callback-style function that internally uses async/await code, we don't currently have an established pattern.
+- To call our own library functions from async/await code, we provide a version of them with "Async" appended to the name:
```

Comment:
If this is kept, it might be worth mentioning that this is only done if there is a need to keep both versions, but in general the callback version should only be kept while it is in use.

Reply:
This section will be deleted.